input/CIB_Lenz.yml

```diff
@@ -0,0 +1,17 @@
+mapper_class: MapperCIBLenz
+map_name: CIB_Lenz
+file_map: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_fullmission.hpx.fits
+file_hm1: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_evenring.hpx.fits
+file_hm2: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_oddring.hpx.fits
+file_mask: /mnt/extraspace/damonge/Datasets/CIB_Lenz/mask_apod.hpx.fits
+gal_mask_mode: '0.2'
+sp_mask_mode:  '545'
+beam_info: [{'type': 'Custom', 'file': '/mnt/extraspace/damonge/Datasets/CIB_Lenz/windowfunctions.csv',
+             'field': 'Wl_eff'},
+             {'type': 'Pixel',
+             'nside_native': 2048,
+             'nside_wanted': 4096}]
+coords: G
+custom_auto: True
```

Comment:
Isn't this hard-coded into anything that inherits from `PlankBase`?
https://github.com/xC-ell/xCell/blob/11eebb08fb6d31a97032e9610e5479745afe08fa/xcell/mappers/mapper_Planck_base.py#L28

Reply:
you are right, removing custom_auto: True from all of Planck_base children